pymatgen/io/lobster/lobsterenv.py

```diff
@@ -1134,10 +1134,10 @@ def _get_limit_from_extremum(
 
         Args:
             icohpcollection: icohpcollection object
-            percentage: will determine which ICOHPs will be considered (only 0.15 from the maximum value)
+            percentage: will determine which ICOHPs/ICOOP/ICOBI will be considered (only 0.15 from the maximum value)
             adapt_extremum_to_add_cond: should the extrumum be adapted to the additional condition
             additional_condition: additional condition to determine which bonds are relevant
-        Returns: [-100000, min(max_icohp*0.15,-0.1)]
+        Returns: [-100000, min(max_icohp*0.15,-0.1)] / [max_icohp*0.15,100000]
```

Comment:
Could we make this -0.1 hard limit a parameter and allow to turn it off? This is my fault

Have you added tests that this part is working correctly?

Reply:
Maybe, we can find a more elegant solution that also works for different units.

Reply:
Now limits returned uses `float('inf') ` instead of hardcoded numbers